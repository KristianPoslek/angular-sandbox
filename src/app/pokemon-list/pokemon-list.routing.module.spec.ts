import { routes } from './pokemon-list.routing.module';

describe('pokemonList.PokemonListRoutingModule', () => {
    it('should define the routes', () => {
        expect(routes).toMatchSnapshot();
    });
});
