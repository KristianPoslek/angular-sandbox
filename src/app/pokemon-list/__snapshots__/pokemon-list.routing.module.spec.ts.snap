// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pokemonList.PokemonListRoutingModule should define the routes 1`] = `
Array [
  Object {
    "component": [Function],
    "path": "",
  },
]
`;
